util/registry_utils.ts

```diff
@@ -245,18 +248,18 @@ export interface Build {
   message?: string;
 }
 
-export async function getBuild(id: string): Promise<Build> {
+export async function getBuild(id: string): Promise<Build | Error> {
   const url = `${API_ENDPOINT}builds/${id}`;
   const res = await fetch(url, { headers: { accept: "application/json" } });
   if (res.status !== 200) {
-    throw Error(
+    return Error(
```

Comment:
no action needed, but note this is not idiomatic javascript

Reply:
well, we want to display the error in the page.